cli/src/commands/log.rs

```diff
@@ -110,18 +111,29 @@ pub(crate) fn cmd_log(
         .settings()
         .config()
         .get_bool("ui.log-synthetic-elided-nodes")?;
-    let template = workspace_command.parse_commit_template(&template_string)?;
     let with_content_format = LogContentFormat::new(ui, command.settings())?;
 
+    let template;
+    let commit_node_template;
+    {
+        let language = workspace_command.commit_template_language()?;
+        template = workspace_command.parse_template(&language, &template_string)?;
```

Comment:
nit: `template_string` can also be moved in this block.

Reply:
Done.